code/modules/projectiles/ammo_datums.dm

```diff
@@ -2576,24 +2576,28 @@ datum/ammo/bullet/revolver/tp44
 	hud_state = "flame"
 	hud_state_empty = "flame_empty"
 	damage_type = BURN
-	flags_ammo_behavior = AMMO_INCENDIARY|AMMO_IGNORE_ARMOR|AMMO_FLAME
+	flags_ammo_behavior = AMMO_INCENDIARY|AMMO_IGNORE_ARMOR|AMMO_FLAME|AMMO_EXPLOSIVE
```

Comment:
What?

Reply:
Explosive ammo tag basically means 'can be aimed at an open turf and will land there instead of continuing past', which seemed like a reasonable and desired thing for it to have. Plasma pistol's the same.